hw/ip/i2c/dv/i2c_sim_cfg.hjson

```diff
@@ -80,6 +80,12 @@
       name: i2c_stretch_timeout
       uvm_test_seq: i2c_stretch_timeout_vseq
     }
+
+    {
+      name: i2c_error_intr
+      reseed: 10
```

Comment:
nit: perhaps don't reduce the reseed for this as it takes time to test all kinds of reset scenarios

Reply:
@weicaiyang i2c_error_intr has a high value of num_trans than the others because I want to increase the chance all error interrupts appear and reset in a single run. This is a trade-off between running a few times of a long test or many times of a short test.

Reply:
I think we may have a better answer when we try to make coverage 100%. We can adjust this later.